packages/ag-charts-community/src/chart/series/seriesAreaManager.ts

```diff
@@ -411,7 +412,15 @@ export class SeriesAreaManager extends BaseManager {
         }
 
         // Update the bounds of the focus indicator:
-        const keyboardEvent = makeKeyboardPointerEvent(hoverRect, this.focusIndicator, pick);
+        drawPickedFocus(
+            seriesRect != null ? new BBox(0, 0, seriesRect.width, seriesRect?.height) : undefined,
+            this.focusIndicator,
+            pick,
+            seriesRect?.x,
+            seriesRect?.y
+        );
+
+        const keyboardEvent = makeKeyboardPointerEvent(hoverRect, pick);
```

Comment:
nit (optional): you could add the `seriesRect` parameter to `makeKeyboardPointerEvent` instead of exporting `drawPickedFocus`.